docs/OV_Runtime_UG/supported_plugins/GPU_RemoteTensor_API.md

```diff
@@ -96,19 +135,31 @@ Once the plugin options have been changed, the internal context is replaced by t
 
 To request the current default context of the plugin, use one of the following methods:
 
+.. tab:: C++
+   .. tab:: Get context from Core
```

Comment:
<img width="738" alt="Screenshot 2023-03-21 at 10 39 33 AM" src="https://user-images.githubusercontent.com/2566854/226534503-ba0d16b3-70dc-4d9a-9c6a-611df011dca2.png">

Not rendered well

Reply:
done